### PR TITLE
FlexBox

### PR DESCRIPTION
**Pull request by Prapti-Patel**
# Pull request

## Implement

- [ ] Update pull request (title, assignee, labels, projects, implement, measure)
- [ ] Update issue numbers here which will be fixed by this PR | Fixes #
- [ ] Update labels for clarity & PR workflow
- [ ] Add/Update comments in code
- [ ] Self code review as per this [guide](https://google.github.io/eng-practices/review/)
- [ ] Add/Update change logs, docs and release notes

## Measure

- [ ] What: Pull Request Title | Why: PR Detail | How: DIM | When: Timeline | Who: Assignee & Reviewer
- [ ] Review change logs, docs & release notes
- [ ] After completing all subtasks validate and make PR ready to merge

### CODE REVIEW

**Comment by Prapti-Patel** [comment]
@beingminimal  Please Review

**Comment by beingminimal** [comment]
@patel-rudra17 Review this PR

**Comment by patel-rudra17** [comment]
Before doing task please refer the proper guidance link  (https://flexboxfroggy.com/) .

**Comment by Prapti-Patel** [comment]
@patel-rudra17 sir, I had completed all the task in this link (https://flexboxfroggy.com/). please review.

File: individual-projects/2023/Prapti-Patel-190770107179/FlexBox/index.html [hunk 1]
```diff
@@ -1,16 +1,27 @@
...
+
 <head>
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <title>CSS Flexbox 01</title>
+
+    <style>
+        #pond {
```
Comment (patel-rudra17) [inline individual-projects/2023/Prapti-Patel-190770107179/FlexBox/index.html:11]: where is the id in your body .....

here you have to use class = "icons"

File: individual-projects/2023/Prapti-Patel-190770107179/FlexBox/index.html [hunk 2]
```diff
@@ -1,16 +1,27 @@
...
-        <img src="https://images.unsplash.com/photo-1614680376408-81e91ffe3db7?ixlib=rb-4.0.3&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1074&q=80" alt="whatsapp">
-        <img src="https://images.unsplash.com/photo-1611262588024-d12430b98920?ixlib=rb-4.0.3&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1074&q=80" alt="instagram">
-        <img src="https://images.unsplash.com/photo-1611605698335-8b1569810432?ixlib=rb-4.0.3&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1074&q=80" alt="twitter">  
-    </div>
+    <a href="https://flexboxfroggy.com/">Flexbox example</a>
 </body>
+
+</html>
+
 </html>
```
Comment (patel-rudra17) [inline individual-projects/2023/Prapti-Patel-190770107179/FlexBox/index.html:27]: why you close html second time.

**Review by patel-rudra17 — CHANGES_REQUESTED** [review]

**Comment by Prapti-Patel** [comment]
@patel-rudra17 I update it sir. Please review

**Review by patel-rudra17 — APPROVED** [review]

**Pull request merged**